server/web/api/register.py

```diff
@@ -69,58 +97,57 @@ def login():
 @api.get("/templates")
 @admin_only
 def get_templates():
-    return [
-        {"id": 10023, "name": "Busunfall", "players": 5},
-        {"id": 900323, "name": "Explosion im Wohnviertel", "players": 10},
-    ]
+    return [{"id": scenario.id, "name": scenario.name, "executions": [execution.id for execution in scenario.executions]}
+            for scenario in models.Scenario.query]
 
 
 @api.post("/scenario/start")
-@admin_only
+# @admin_only
+@csrf.exempt
 def start_scenario():
     try:
-        id = request.form["id"]
+        id = int(request.form["id"])
     except KeyError:
         return {"error": "Missing id in request"}, 400
 
-    # TODO: Create actual execution
-    exec_id = math.floor(random() * 1000)
-    test_execution.update({
-        "id": exec_id,
-        "status": "",
-        "players": [
-            {
-                "tan": x,
-                "name": "Max Mustermann",
-                "status": choice(["", "In Vorbereitung"]),
-                "action": "Legt Zugang",
-            }
-            for x in [str(x) for x in uniques(5)]
-        ],
-    })
-    return {
-        "id": exec_id,
-    }
+    if load_execution(id):
+        return active_executions[id].to_dict()
+
+    # Failure
+    raise Exception(f"Could not load execution with id {id}")
 
 
 @api.get("/execution/<int:id>")
 @admin_only
 def get_execution_status(id: int):
-    test_execution["id"] = id
-    random_player = choice(test_execution["players"])
-    random_player["status"] = "In Vorbereitung" if random_player["status"] == "" else ""
-    return test_execution
+    execution = try_get_execution(id)
+    return execution.to_dict()
 
 
 @api.post("/execution/<int:id>/start")
 @admin_only
 def start_execution(id: int):
-    test_execution["status"] = "Aktiv"
-    return test_execution
+    execution = try_get_execution(id)
+    execution.status = Execution.Status.RUNNING
+    return Response(status=204)
```

Comment:
why no content instead of 200? Based on the state change i think 200 fits better.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
That is because I implemented them ^^. Overall the status indicates a valid request but no changes in the backend due to missing content. 
- app: The methods blocks every run request from any mobile device to prevent game manipulation before the action has started. 
- lobby: "The execution has not been started yet" - same reason here.
- test_lobby is a test case for the lobby implementation and has no further impact on the app.